applications/nrf5340_audio/src/modules/lc3_file.c

```diff
@@ -12,6 +12,34 @@ LOG_MODULE_REGISTER(sd_card_lc3_file, CONFIG_MODULE_SD_CARD_LC3_FILE_LOG_LEVEL);
 
 #define LC3_FILE_ID 0xCC1C
 
+static void lc3_header_print(struct lc3_file_header *header)
+{
+	if (header == NULL) {
+		LOG_ERR("Nullptr received");
```

Comment:
Nitpick but not important: since in the later part I see we use "null pointer" instead of "nullptr"
We can update in here too, but not very necessary.